### ISSUE TITLE
Trackpad not working

### ISSUE THREAD

**Issue by sebpalluel** [gnodipac886/MatebookXPro-hackintosh#6]
Hello,

First thank you very much for your amazing work !

I have an issue with the trackpad that is not recognized in system preference. The kexts `VoodooI2C.kext` and `VoodooI2CHID.kext` are indeed installed in `/EFI/CLOVER/kexts/OTHER`. I've succeeded to apply the DSDT patch you provided for the brightness keys (i've had two error on compile in MaciASL so i removed the section with issues to be able to save) but i can't find the patch for the trackpad that you say is required on your last update. Beside of that everything seems to work like a charm.

**Comment by gnodipac886** [comment]
Hi, you would need to apply the SKL+ patch as well as the Windows10 patch. You can just use the MaciASL that I attached in the tools.zip folder. While patching, the patches are all located on the left hand side of the smaller patching window. Please try that.

**Comment by sebpalluel** [comment]
Thanks, the issue is resolved by using the MaciASL from your repo and applying the two patches on my patched DSDT.aml in `EFI/CLOVER/ACPI/patched/`.
Tho i noticed that sometime when the lid is closed for a while, the computer get a kernel panic.

Also just to inform you, your topic in tonyx86 is not accessible: 
<img width="1182" alt="Screen Shot 2019-05-04 at 12 47 53 PM" src="https://user-images.githubusercontent.com/11297176/57178031-fbb45900-6e6a-11e9-902c-e4f78c40ba98.png">

**Comment by adoster** [comment]
This issue appeared just in time -- I installed Mojave 10.14.4 last night on my 2018 i7 MXP and was specifically looking into why trackpad/brightness still wasn't working.

I'm not sure if either of you ran into this issue, but my default DSDT had errors related to `PAFS`. I resolved it (so as of this moment, touchpad and brightness works!) by simply commenting it out.

Errors:
```
35219, 6086, Not a control method, cannot invoke (PAFS is a BufferField)
35219, 6058, Invalid type (Target is [Package], Source must be a package also)
```

"Fix:"
```
//    Method (PAFS, 1, NotSerialized)
//    {
//        Name (BUFF, Buffer (0x0100){})
//        Store (Arg0, Local0)
//        CreateByteField (BUFF, Zero, STAT)
//        CreateByteField (BUFF, One, PAFS)
//        Store (PAFL, PAFS (Store (Zero, STAT)))
//        Return (BUFF)
//    }
```

I tried googling for PAFS-related DSDT issues and only got somewhat relevant answers. Also, I'm not an experienced Hackintosher. As such, I have no idea what I just commented out. Do either of you know whether this is advisable or not/have a better solution that actually edits the syntax? Much appreciated.

Also, I would like to say thanks for all your work gnodipac886. You and Darren have made it a very seamless experience for the community to use MacOS on our Matebook X Pros.
